snippets/make.snippets

```diff
@@ -1,3 +1,28 @@
+# base
+snippet base
+	.PHONY: clean, mrproper
+	CC = gcc
+	CFLAGS = -g -Wall
+
+	all : $1
```

Comment:
I never saw Makefile rules separated with a space from the condition. I.e. `all: $1` instead of `all : $1`. Could you fix that throughout the file?